core/pom.xml

```diff
@@ -76,11 +76,22 @@
         </dependency>
 
         <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
+        <dependency>
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
             <version>${json.org.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts</artifactId>
+            <version>1.13</version>
+        </dependency>
```

Comment:
Will check, because I need harmonized versions between this place and the GeoJSON serializer. But that one has a newer version, too, so...

BTW are you fine, Android-wise, with jts-core in the GraphHopper core? We were worrying..